website/docs/r/cognito_user_pool_domain.markdown

```diff
@@ -34,9 +34,23 @@ resource "aws_cognito_user_pool_domain" "main" {
 resource "aws_cognito_user_pool" "example" {
   name = "example-pool"
 }
-```
 
+data "aws_route53_zone" "example" {
+  name = "example.com"
+}
 
+resource "aws_route53_record" "auth-cognito-A" {
+  name    = "${aws_cognito_user_pool_domain.domain}"
+  type    = "A"
+  zone_id = "${data.aws_route53_zone.example.id}"
```

Comment:
I often see `data` used in this way. For me, I needed to change this:

```suggestion
  zone_id = "${aws_route53_zone.example.zone_id}"
```

Reply:
I detected that I was adding data.

> I often see `data` used in this way. For me, I needed to change this:

I need to revert this. Cause in the lines above I referenced data.aws_route53_zone...